src/stacking.rs

```diff
@@ -0,0 +1,53 @@
+
+use imp_prelude::*;
+use error::{ShapeError, ErrorKind, from_kind};
+
+/// Stack arrays along the given axis.
+pub fn stack<'a, A, D>(arrays: &[ArrayView<'a, A, D>], axis: Axis)
```

Comment:
This now seems best with Axis first and then arrays. Many methods use `Axis` as first argument now, so it seems fine.